packages/libs/react-ui/src/styles/atoms.css.ts

```diff
@@ -0,0 +1,130 @@
+import { createSprinkles, defineProperties } from '@vanilla-extract/sprinkles';
+import mapValues from 'lodash.mapvalues';
+import { breakpoints } from './themeUtils';
+import { tokens } from './tokens/contract.css';
+import { darkTheme } from './tokens/dark.css';
+import { lightTheme } from './tokens/light.css';
+import { flattenTokens } from './tokens/utils';
+
+const systemProperties = defineProperties({
+  properties: {
+    background: ['none'],
+    border: {
+      none: 'none',
+      hairline: tokens.kda.foundation.border.hairline,
+      normal: tokens.kda.foundation.border.normal,
+      thick: tokens.kda.foundation.border.thick,
+    },
+    borderRadius: tokens.kda.foundation.radius,
+    borderWidth: tokens.kda.foundation.border.width,
+    bottom: [0],
+    boxShadow: tokens.kda.foundation.effect.shadow,
+    cursor: ['pointer', 'not-allowed'],
+    flex: [1],
+    flexGrow: [0, 1],
+    flexShrink: [0],
+    flexWrap: ['wrap', 'nowrap'],
+    fontFamily: tokens.kda.foundation.typography.family,
+    fontWeight: flattenTokens(tokens.kda.foundation.typography.weight),
+    height: ['100%'],
+    inset: [0],
+    left: [0],
+    listStyleType: ['none'],
+    maxWidth: {
+      'content.maxWidth': tokens.kda.foundation.layout.content.maxWidth,
+    },
+    minWidth: {
+      'content.minWidth': tokens.kda.foundation.layout.content.minWidth,
+    },
+    opacity: [0, 1],
+    outline: ['none'],
+    overflow: ['hidden', 'visible', 'scroll', 'auto'],
+    overflowX: ['hidden', 'visible', 'scroll', 'auto'],
+    overflowY: ['hidden', 'visible', 'scroll', 'auto'],
+    pointerEvents: ['none'],
+    position: ['fixed', 'static', 'absolute', 'relative', 'sticky'],
+    right: [0],
+    textAlign: ['left', 'center', 'right'],
+    textDecoration: ['underline', 'none'],
+    textTransform: ['uppercase', 'lowercase', 'capitalize', 'none'],
+    top: [0],
+    whiteSpace: ['nowrap', 'break-spaces', 'normal', 'pre-wrap'],
+    width: ['100%'],
+    wordBreak: ['normal', 'keep-all', 'break-word', 'break-all'],
+    zIndex: [-1, 0, 1],
+  },
+});
+
+const colorProperties = defineProperties({
+  conditions: {
+    lightMode: { selector: `.${lightTheme} &` },
+    darkMode: { selector: `.${darkTheme} &` },
+  },
+  defaultCondition: 'lightMode',
+  properties: {
+    color: flattenTokens({
+      icon: tokens.kda.foundation.color.icon,
+      text: tokens.kda.foundation.color.text,
+    }),
+    backgroundColor: {
+      transparent: 'transparent',
+      ...flattenTokens(tokens.kda.foundation.color.background),
+    },
+    borderColor: flattenTokens(tokens.kda.foundation.color.border),
+  },
+});
```

Comment:
@isaozler and @salamaashoush should we move these to the `systemProperties` since they should already automatically switch depending on the theme?